src/assets/DockerCompose/docker_compose.md

```diff
@@ -0,0 +1,211 @@
+      Here, we have another keyword `depends_on` which has value `database` service, that means that `backend` service is only ran after `database` successfully run.
+
+  3. frontend service
+
+      ```yml
+          frontend:
+            image: sagar4321/demo-web
+            environment:
+              SERVER_URL: http://0.0.0.0:3001
+            ports:
+              - 8080:8080
+            depends_on:
+              - backend
+      ```
+
+      Same definition and configuration for the `frontend` service with `image: sagar4321/demo-web` (which I built and pushed to docker hub), `environment` and `port 8080` (where the `frontend` service listen).
+      But here the `frontend` service depends on `backend` service since it needs to be started before the `frontend` service.
+
+  4. phpmyadmin service
+
+      ```yml
+        phpmyadmin:
+          image: phpmyadmin/phpmyadmin
+          restart: always
+          environment:
+            PMA_HOST: database
+            PMA_PORT: 3306
+          ports:
+            - "8081:80"
+          depends_on:
+            - database
+      ```
+     Also, same thing applies defining and configuring `phpmyadmin` services. It also depends on `database` services. And listen to `port:8081`
+
+Now the overall `compose.yml` file after combining all the services together looks something like this as one:
+
+```yml
+version: "3.8"
+
+services:
+  database:
+    image: mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: demodb
+    volumes:
+      - ./testdump.sql:/docker-entrypoint-initdb.d/testdump.sql
+    restart: always
+
+  server:
+    image: sagar4321/demo-server
+    environment:
+      PORT: 3001
+      DBHOST: database
+      DBPORT: 3306
+      DBUSER: root
+      DBNAME: demodb
+      DBPASSWORD: root
+    restart: always
+    ports:
+      - "3001:3001"
+    depends_on:
+      - database
+
+  web:
+    image: sagar4321/demo-web
+    environment:
+      SERVER_URL: http://0.0.0.0:3001
+    ports:
+      - 8080:8080
+    depends_on:
+      - server
+
+  phpmyadmin:
+    depends_on:
+      - database
+    image: phpmyadmin/phpmyadmin
+    restart: always
+    environment:
+      PMA_HOST: database
+      PMA_PORT: 3306
+    ports:
+      - "8081:80"
+
+```
+
+Yes, we have finally completed setting up our required services for our application through the compose file `compose.yml`
+Now all it takes is the below single command to run the whole services in multiple containers and get up our application running.
+From the directory `dockercompose` in the terminal just run the command as:
+```console
+docker-compose -f compose.yml up
+```
+
+The command get up all the services running together, just wait a while so that all those services get up running, then we can
```

Comment:
```suggestion
The command starts up all the services running together. Just wait a while so that all those services get up and running, then we can
```